client/pokemon/pokemon-full.view.html

```diff
@@ -1,12 +1,315 @@
+                <span flex="35">{{move.moveName || '-'}}</span>
+                <span flex="25">Type</span>
+                <span flex>Power</span>
+                <span flex="10">{{move.Pp || '-'}}</span>
+                <span flex="10">{{move.Ppu || '-'}}</span>
+              </div>
+              <hr />
+              <div>
+                <strong>Relearnable</strong>
+                <div><span flex="25" ng-repeat="move in pokemon.eggMoves track by $index">{{move || 'N/A'}}{{$last ? '' : ', '}}</span></div>
+              </div>
+            </md-card-content>
+          </md-card>
+        </div>
+
+        <div layout-gt-xs="column">
+          <md-card class="summary" ng-show="pokemon.hasFullData">
+            <md-card-header><span class="md-headline">Encounter Info</span></md-card-header>
+            <md-card-content>
+              <div>
+                <strong>Game of Origin</strong>
+                <span ng-class="pokemon.gameLabel">{{pokemon.data.otGameName}}</span>
+              </div>
+              <div ng-if="pokemon.data.isFatefulEncounter">Fateful Encounter</div>
+              <div ng-if="pokemon.data.eggLocationId == 0">
+                Met at <span>{{pokemon.data.metLocationName}}</span>
+              </div>
+              <div>
+                Met at level {{pokemon.data.levelMet}} on {{pokemon.displayMetDate}}
+              </div>
+              <div ng-if="pokemon.data.eggLocationId != 0">
+                <div>Egg received from <span>{{pokemon.data.eggLocationName}} on {{pokemon.displayEggDate}}</span></div>
+                <div>Hatched at <span>{{pokemon.data.metLocationName}}</span></div>
+              </div>
+            </md-card-content>
+          </md-card>
+
+          <md-card class="summary" ng-show="pokemon.hasFullData">
+            <md-card-header><span class="md-headline">Region Info</span></md-card-header>
+            <md-card-content>
+              <div><strong>Language</strong> <span>{{pokemon.data.language}}</span></div>
+              <div><strong>Country</strong> <span>{{pokemon.data.countryId}}</span></div>
+              <div><strong>Subregion</strong> <span>{{pokemon.data.regionId}}</span></div>
+              <div><strong>3DS Region</strong> <span>{{pokemon.data.consoleRegion}}</span></div>
+            </md-card-content>
+          </md-card>
+
+          <md-card class="summary" ng-show="pokemon.hasFullData">
+            <md-card-header><span class="md-headline">Upload Info</span></md-card-header>
+            <md-card-content>
+              <!-- add link to owner's profile -->
+              <div><strong>Owner</strong> {{pokemon.data.owner}}</div>
+              <div><strong>Unique in Porybox database?</strong>
+                <span ng-if="pokemon.data.isUnique">Yes</span>
+                <span ng-if="!pokemon.data.isUnique">No</span>
+              </div>
+              <div><a ng-href="/#/box/{{pokemon.data.box}}">Back to box</a></div>
```

Comment:
Be sure to account for the case where the viewer is not the pokemon's owner; `pokemon.data.box` will be undefined.